### ISSUE TITLE
Source Code

### ISSUE THREAD

**Issue by robotnikthingy** [Shampaggon/CrackShot#27]
Can you please consider releasing the source code for this plugin since it looks like its no longer in development?

Crackshot is basically the de-facto weapons plugin for bukkit/spigot with no alternatives, and if we had access to the source code I would have no doubt people would help contribute to its development due to its popularity (such as the creator of CrackShotPlus, adding more API methods).

**Comment by robotnikthingy** [comment]
hello?

**Comment by noahcoetsee** [comment]
Check this out: https://github.com/Shampaggon/CrackShot/wiki/Hooking-into-CrackShot

**Comment by noahcoetsee** [comment]
And if you want Source Code, just unzip the jar file...

**Comment by engineer1109** [comment]
Do you uncompile successfully?

**Comment by noahcoetsee** [comment]
No, however, clearly he doesn't want somebody just copying his code, so just hook in or use a different gun plugin..
